pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp.md

```diff
@@ -0,0 +1,553 @@
+    <img src="https://www.example.com/assets/cover1.html" loading="lazy" width="100%" height="100%" data-amp-story-player-poster-img>
+    A title that describes this story.
+  </a>
+</amp-story-player>
+```
+
+
+For the best user experience, we strongly recommend including a poster image. If you do not include a poster image the story player will display a loader spinner with a grey background.
+
+
+## Specify multiple Web Stories
+
+Include the desired number of `<a>` tags, with each `href` attribute pointed to the desired Web Story URL, within the `<amp-story-player>` element.
+
+[example preview="top-frame" playground="true"]
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script
+      async
+      src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+      href="https://cdn.ampproject.org/amp-story-player-v0.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <style>
+      header {
+        height: 8vh;
+        color: #545454;
+        background-color: #DDB556;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      amp-story-player {
+        margin: 1rem auto;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Page Header
+      </h1>
+    </header>
+    <h1>
+      Article Title
+    </h1>
+    <amp-story-player style="width: 370px; height: 622px;">
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s1"></a>
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s2"></a>
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s3"></a>
+    </amp-story-player>
+  </body>
+</html>
+```
+[/example]
+
+
+The Web Story Player displays the first embedded story. It automatically presents the user with the next story’s cover page after finishing the previous, or using the swiping motion to skip. The Web Story Player does not have any built-in UI functionality to inform the user there are multiple stories embedded in a single player. Instead, build [user entry points](#user-entry-points) to display available stories and allow user selection.  
+
+
+### Circular story wrapping
+
+You can create a circular consumption of a set of stories by adding the `circular-wrapping` action to the JSON configuration. When `circular-wrapping` is included, the Web Story Player displays the first story to users after they finish the last story.
+
+```html
+<amp-story-player>
+ <script type="application/json">
+   {
+     "behavior": {
+       "on": "end",
+       "action": "circular-wrapping"
+     }
+   }
+ </script>
+ <a href="./story1.html"> ... </a>
+ <a href="./story2.html"> ... </a>
+  ...
+</amp-story-player>
+``` 
+
+The JSON configuration must be a direct child of the `<amp-story-player>` element and include the `type="application/json"` attribute. 
+
+
+### Infinite stories
```

Comment:
For this section, @raovs was suggesting planting the idea of personalization when fetching more stories. Could we add a line here that mentions that publishers should be able to create personalization systems on their backends and use the `fetch` action of the player to serve personalized stories based on user behavior?

We should also remind users that the player dispatches events on user behaviors and are linked [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md#custom-events) or the section below for custom events.

Reply:
Added a line! LMKWYT and if theres any guidance we can include about setting up the backend system. Or is that something extremely dependent on the existing backend?

Reply:
That would depend on their existing backend, so we don't wanna jump into that.